Pilot/tests/.pytest_cache/v/cache/lastfailed

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Is this file necessary?

Reply:
I am not familar with this pytest cache so probably the answer is yes

Reply:
No, `.pytest_cache` is a temporary folder created by pytest, none of the files should be kept inside git. (There is even a README file in this folder which says so...)